modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -73,7 +73,7 @@
 	alt_titles = list("Detective", "Forensic Technician", "Private Investigator", "Forensic Scientist")
 
 /datum/job/doctor
-	alt_titles = list("Medical Doctor", "Surgeon", "Nurse", "General Practitioner", "Medical Resident", "Physician")
+	alt_titles = list("Medical Doctor", "Surgeon", "Nurse", "General Practitioner", "Medical Resident", "Physician", "Medical Intern")
```

Comment:
This is the only one I'm not really a fan of, intern roles I feel should be handled ICly and are more likely to be used as an excuse not to do work

Reply:
Alright, I can fix that.

Reply:
Alright, should be fixed now.